src/ut/handlers_test.cpp

```diff
@@ -1782,6 +1782,41 @@ TEST_F(HandlersTest, DigestIMPUNotFound)
   t->on_failure(&mock_op);
 }
 
+TEST_F(HandlersTest, DigestNoIMPUCacheConnectionFailure)
+{
+  // This test tests an Impi Digest task case where no public ID is specified
+  // on the HTTP request, the cache request fails. Start by building the HTTP
```

Comment:
Worth a comment at the top to explain the difference between this and DigestNoIMPUCacheFailure?